api/pom.xml

```diff
@@ -90,7 +91,7 @@
         <api_package>javax.enterprise.concurrent</api_package>
         <non.final>false</non.final>
         <last.spec_version>0.0</last.spec_version>
```

Comment:
Could you check to see if the last.spec_version also needs to be updated? Thanks

Reply:
"last.spec_version" is not being used by plugin. Updating this will not have any effect.